benchmark/kmeans.py

```diff
@@ -52,7 +46,7 @@ def __init__(
         self.results = None
         self.skip_slow = skip_slow
 
-    def timeit(self, kmeans_single_lloyd_fn, name, slow=False):
+    def timeit(self, name, engine_provider=None, slow=False):
```

Comment:
`engine_provider` must be added to the docstring.

Super minor nitpick: what about renaming `slow` to `is_slow`?